src/harness/reference_models/fss_purge/testdata/fss_record_0.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "incumbent/ibfs/0",
+  "type": "FSS",
+  "deploymentParam": 
+  [
+    {
+      "installationParam": {
+        "latitude": 38.985278,
+        "longitude":  -77.445833,
+        "height": 26.5,
+        "elevation": 100,
+        "heightType": "AGL",
+        "antennaGain": 0,
+        "antennaBeamwidth": 0,
+        "antennaAzimuth": 270,
+        "antennaDowntilt": 0,
+        "indoorDeployment": true,
+        "antennaElevationAngle":39
+      },
+      "operationParam": {
+        "operationFrequencyRange": {
+          "lowFrequency": 3610000000,
+          "highFrequency": 4200000000
+        }
+      },
+      "ttc": true,
```

Comment:
This (and the two parameters below) is not part of the SAS Testing API. Do we need to add it?

Reply:
These   two parameters have been added as part of this submission into the fss   records.

Reply:
Please propose this change to the testing API via the test dev mailing list so that everyone is aware and can comment.

Reply:
You will also need to update the existing FSS test data. It's probably best to do that in a separate PR so that it can get submitted quickly.

Reply:
Agreed. We have submitted a new PR #486 for this. Please review.